src/course-unit/utils.ts

```diff
@@ -28,3 +28,18 @@ export const adoptCourseSectionUrl = (
 
   return newUrl;
 };
+
+/**
+ * Generates the edit URL for the first unit of a given subsection in a course.
+ *
+ * @param {Object} params - The parameters required to build the URL.
+ * @param {string} params.courseId - The ID of the course.
+ * @param {string} params.subsectionId - The ID of the subsection.
+ * @returns {string} The constructed edit URL for the subsection's first unit.
```

Comment:
```suggestion
 * @returns The constructed edit URL for the subsection's first unit.
```
You don't have to include the type information when adding JSDoc comments to TypeScript files - it's already specified below.